src/vmm/src/device_manager/persist.rs

```diff
@@ -723,6 +724,9 @@ mod tests {
                 uds_path: tmp_sock_file.as_path().to_str().unwrap().to_string(),
             };
             insert_vsock_device(&mut vmm, &mut cmdline, &mut event_manager, vsock_config);
+            // Add an entropy device.
+            let entropy_config = EntropyDeviceConfig {};
```

Comment:
```suggestion
           let entropy_config = EntropyDeviceConfig;
```

Reply:
Ok, so I applied this change (here and all other places you spotted) but it breaks some unit-tests. Apparently when you serialize to json `EntropyDeviceConfig` results to `None` whereas `EntropyDeviceConfig {}` to `{}`.

Since in the next PR I will populate this struct what I did was define the struct like this:

```
#[derive(Default)]
struct EntropyDeviceConfig {}
```

and the use `EntropyDeviceConfig::default()` which is slightly nicer than `EntropyDeviceConfig {}`